docs/6.x/changelog.md

```diff
@@ -38,6 +38,23 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 6.x Releases
 
+### 6.1.6 (October 10th, 2019)
+
+#### Improvements
+
+* Improves `gravity report` to capture additional network configuration.
```

Comment:
I've started linking to PRs in changelog (and so did Roman), something along the lines of: https://github.com/gravitational/gravity/pull/657/files#diff-7f6b34363842a7bc7817d07c7ee85bbbR311 - could you please link them as well?